### PR TITLE
Improve descriptions of section headers

### PR DESCRIPTION
**Pull request by chapmanb**
Some of the output section headers we're a bit too technical and require
knowledge of the individual programs (ie you need to know samtools stats
deals with alignments). This tries to make some small improvements to
make MultiQC reports more accessible to biologists with less tool experience.

### CODE REVIEW

**Comment by ewels** [comment]
This is great, thanks! In fact, you've touched on something that I'd like to do across the board for all modules - improve the names and especially the descriptions above each plots to explain what they're showing. They are all a bit minimal currently, fine if you're already familiar with the program, but not ideal if you're not a bioinformatician and someone sends you a report.

I might make some comments with a few alternative suggestions if that's ok, as some of your names are quite long and I prefer having short titles in the side navigation if possible:

image

Thanks for kickstarting this, I'll make a new issue for the general all-modules task.

File: multiqc/modules/snpeff/snpeff.py [hunk 1]
```diff
@@ -44,23 +44,23 @@ def __init__(self):
         # Report sections
         self.sections = list()
         self.sections.append({
-            'name': 'Counts by Genomic Region',
+            'name': 'Variant Counts by Genomic Region',
```
Comment (ewels) [inline multiqc/modules/snpeff/snpeff.py]: Perhaps just `Variants by Genomic Region`?

File: multiqc/modules/samtools/stats.py [hunk 2]
```diff
@@ -128,7 +128,7 @@ def parse_samtools_stats(self):
             plot_html = plots.beeswarm.plot(self.samtools_stats, keys,
                                             {'id': 'samtools-stats-dp'})
             self.sections.append({
-                'name': 'Samtools Stats',
+                'name': 'Alignment metrics from samtools stats',
```
Comment (ewels) [inline multiqc/modules/samtools/stats.py]: Maybe just `Alignment metrics`?

Comment (ewels) [inline multiqc/modules/samtools/stats.py]: Maybe also the other section in this module as `Percent Mapped` instead of `Alignment`?

**Comment by ewels** [comment]
See #289 for the general issue.

**Comment by chapmanb** [comment]
Phil -- thanks so much for the helpful feedback. I shortened these based on your suggestions and happy to help with the more long term project of improving explanations for each tool.

**Comment by ewels** [comment]
Great, thanks!

**Pull request merged**